class/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permissions: true)
+    @name = name
+    @age = age
+    @parent_permissions = parent_permissions
+  end
+
+  def can_use_services?
+    is_of_age? || parent_permissions
+  end
```

Comment:
- [ ] In this method `can_use_services?`, you are referring to the instance variable `@parent_permissions` and not a local variable in the method called parent_permissions. Kindly work on this. 👍

Reply:
👍🏽